models/sepc/builder.py

```diff
@@ -0,0 +1,126 @@
+import mxnet as mx
+import mxnext as X
+from models.NASFPN.builder import RetinaNetNeck, RetinaNetNeckWithBN
+from models.sepc.sepc_neck import SEPCFPN
+from utils.patch_config import patch_config_as_nothrow
+from models.NASFPN.builder import RetinaNetHeadWithBN
+import math
+
+
+class RetinaNetNeckWithBNWithSEPC(RetinaNetNeckWithBN):
+    def __init__(self, pNeck, pSEPC):
+        super().__init__(pNeck)
+        self.psepc = patch_config_as_nothrow(pSEPC)
+        self.neck_with_sepc = None
+        stride, pad_sizes = pSEPC.stride, pSEPC.pad_sizes
+        self.feat_sizes = [[math.ceil(pad_sizes[0]/stride[i]), math.ceil(pad_sizes[1]/stride[i])] for i in range(len(stride))]
+
+    def get_retinanet_neck(self, data):
+        if self.neck_with_sepc is not None:
+            return self.neck_with_sepc
+        fpn_outs = super().get_retinanet_neck(data)
+        p3_conv, p4_conv, p5_conv, p6, p7 = fpn_outs['stride8'], fpn_outs['stride16'], fpn_outs['stride32'], fpn_outs['stride64'], fpn_outs['stride128']
+
+        # add SEPC module after default FPN
+        sepc_inputs = [p3_conv, p4_conv, p5_conv, p6, p7]
+        sepc_outs = SEPCFPN(
+            sepc_inputs,
+            out_channels=self.psepc.out_channels,
+            pconv_deform=self.psepc.pconv_deform,
+            iBN=self.psepc.ibn if self.psepc.ibn is not None else None,
+            Pconv_num=self.psepc.pconv_num,
+            start_level=self.psepc.start_level or 1,
+            norm=self.psepc.normalizer,
+            lcconv_deform=self.psepc.lcconv_deform if self.psepc.lcconv_deform is not None else None,
```

Comment:
Just `self.psepc.lcconv_deform or False` is ok, the `patch_config_as_nothrow ` make the config class return None if a config entry does not exist.
Change bilinear_upsample accordingly.

Additionally, do not mix False and None, None has very different semantic of False and we already use the None semantic to represent missing attribute in a config class.